commons-geometry-core/src/test/java/org/apache/commons/geometry/core/partitioning/AbstractConvexHyperplaneBoundedRegionTest.java

```diff
@@ -33,6 +33,8 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Static imports are fine IMO, but stylistically, the library tests are written `Assertions.assert*`. So, we should keep it consistent.

Reply:
Agreed. For consistency, we should use the qualified `Assertions.assertThrows(...)` instead of just `assertThrows(...)`. This convention is used in commons-math and commons-numbers as well.